src/engraving/libmscore/slur.cpp

```diff
@@ -978,6 +987,16 @@ Slur::Slur(Score* s)
     setAnchor(Anchor::CHORD);
 }
 
+//---------------------------------------------------------
+//   calcStemArrangement
+//---------------------------------------------------------
```

Comment:
when copying/pasting slurs around the score, we need to know if the stem arrangement is compatible, or the bezier points don't make sense